packages/mobile/src/app/reducers.ts

```diff
@@ -21,6 +21,7 @@ export interface State {
   inviteModalVisible: boolean
   activeScreen: Screens
   hideVerification: boolean
+  showRaiseDailyLimit: string | undefined
```

Comment:
nit: Would there be a better name, given how it's used?
Maybe `showRaiseDailyLimitTarget`, or `showRaiseDailyLimitSegment`?